pkg/move/copy.go

```diff
@@ -7,14 +8,21 @@ import (
 	"golang.org/x/sys/unix"
 )
 
-type copyFunc func(log logr.Logger, fs afero.Afero, from, to string) error
+const (
+	// defaultUmask is the umask value to use for file operations.
+	defaultUmask = 0000
```

Comment:
IMO better name here based on semantic of what these number mean is `noPermissionsMask`
> umask in Unix-like systems, specifies that newly created files and directories should have no permissions revoked